verifiers/internal/gha/provenance.go

```diff
@@ -179,12 +179,18 @@ func verifyDigest(prov slsaprovenance.Provenance, expectedHash string) error {
 	}
 
 	// 8 bit represented in hex, so 8/2=4.
-	l := len(expectedHash) * 4
+	bitLength := len(expectedHash) * 4
+	expectedAlgo := fmt.Sprintf("sha%v", bitLength)
+	// sha1 is 160 bit (FWIW).
```

Comment:
curious: why do we want to support this algo (which is not considered secure)?

Reply:
Well, the alternative is to support "sha160" which isn't a name anyone uses. Should we support a minimum bit length?

Reply:
Added an issue and TODO comment to track: https://github.com/slsa-framework/slsa-verifier/issues/630